include/jsonv/optional.hpp

```diff
@@ -32,8 +32,13 @@
 #       undef  JSONV_OPTIONAL_USE_STD
 #       define JSONV_OPTIONAL_USE_STD 1
 #   else
-#       undef  JSONV_OPTIONAL_USE_STD_EXPERIMENTAL
-#       define JSONV_OPTIONAL_USE_STD_EXPERIMENTAL 1
+#       if defined(_MSC_VER)
+#           undef  JSONV_OPTIONAL_USE_BOOST
```

Comment:
Tiny thing -- I never want to "magic" a Boost dependency into the project. If you're using MSVC, you need to add the definition manually.